vue/mixins/scroll.js

```diff
@@ -54,7 +79,8 @@ export const scrollMixin = {
         scrollToTop() {
             window.scroll({
                 top: 0,
-                left: 0
+                left: 0,
+                behavior: "smooth"

```

Comment:
This seems like a breaking change so we should avoid it, no?

Reply:
I don't think it's a breaking change in the way that it could potentially break something for whoever's using.
If behavior smooth is not available, it will still scroll not smoothly